### PR TITLE
reset password with sending email

### PR DESCRIPTION
**Pull request by Moradi66**

### CODE REVIEW

File: backend/api/CheckUserToken.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+const CheckUserToken = (req, res) => {
+  const token = req.params.token;
+  var sql =
+    "select resetPasswordToken, resetPasswordExpires, userName from users where resetPasswordToken=?";
+  db.all(sql, [token], (err, rows) => {
+    const [user] = rows;
+    if (!user) {
+      return res.send("Sorry this link is Wrong");
```
Comment (KrustyC) [inline backend/api/CheckUserToken.js]: Here  I would set a state of 403 as basically an error happen and the frontend should know it staright away tithout checking the message

File: backend/api/ForgotPassword.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+var nodemailer = require("nodemailer");
+var crypto = require("crypto");
+
+const sqlite3 = require("sqlite3").verbose();
+
+const filename = "./database/crisisdb.sqlit";
+let db = new sqlite3.Database(filename);
+
+const ForgotPassword = (req, res, next) => {
+  console.log(req.body);
```
Comment (KrustyC) [inline backend/api/ForgotPassword.js]: please remove this

Comment (Moradi66) [reply backend/api/ForgotPassword.js]: you mean console log or?....

File: backend/api/CheckUserToken.js (hunk 1 above)
Comment (Moradi66) [reply backend/api/CheckUserToken.js]: I have problem with set state and getting it in front-end, if you can be in CYF this Sunday it would be great.

**Pull request merged**